README.md

```diff
@@ -1,24 +1,258 @@
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Only Admin can add or delete Cars from the website.
+- Users can create accounts and log in securely to access the booking functionalities.
+- Choose from a wide range of available cars based on your preferences and requirements.
+- Easily book a car for your desired date and time and manage your bookings efficiently.
+- User can Cancel the booking.
+- Website is responsive and is designed to work nicely for different devices and screens.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- React Frontend -->
+
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
+
+- Frontend part of the project [Fast-Lane-Racers-frontend](https://github.com/muneeb3123/Fast-Lane-Racers-front-end)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor (Vs Code)
+- Terminal
+- Install ruby on your computer
+- Install rails on your computer
+- Node.js
+- PostgreSQL
+
+### Setup
+
+Clone these repositories by using the command line to your desired folder:
+
+<u>**For Backend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-back-end
+
+cd Fast-Lane-Racers-back-end
+```
+
+<u>**For Frontend**</u>
+
+```sh
+git clone https://github.com/muneeb3123/Fast-Lane-Racers-front-end
+
+cd Fast-Lane-Racers-front-end
+```
+
+### Install
+
+Install this project with:
+
+<u>**For Backend**</u>
+
+```sh
+bundle install
+```
+
+<u>**For Frontend**</u>
+
+```sh
+npm install
+```
+
+### Usage
+
+<u>**For Backend**</u>
+
+If you are a Microverse reviewer, create a ```master.key``` file in ```config``` directory and paste the master key provided with the review request. If you are someone other than a Microverse Reviewer, contact us.
+
+To run the project, execute the following commands:
+
+```sh
+  rails db:create
+  rails db:migrate
+  rails db:seed
+  rails server
+```
+
+<u>**For Frontend**</u>
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  Backend:  rspec
```

Comment:
- [ ] When I run the tests according to instructions, 1 is failing on my end:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/assets/57421823/8d2f8b5e-d42e-487e-9bfe-960c69767e09)

Reason:  Color can't be nil.
Solution: Just add a color attribute here:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/assets/57421823/ef797cc3-4d20-4231-9cfe-d977a0a8bb4e)

That way, the test will pass (tested). 👍